metron-platform/metron-enrichment/metron-enrichment-common/src/main/java/org/apache/metron/enrichment/stellar/SimpleHBaseEnrichmentFunctions.java

```diff
@@ -132,70 +147,91 @@ private static synchronized void initializeProvider( Map<String, Object> config)
           ,returns = "True if the enrichment indicator exists and false otherwise"
           )
   public static class EnrichmentExists implements StellarFunction {
+
     boolean initialized = false;
-    private static Cache<Table, EnrichmentLookup> enrichmentCollateralCache = CacheBuilder.newBuilder()
-                                                                                        .build();
+    private static Cache<Table, EnrichmentLookup> lookupCache = createCache();
```

Comment:
Altering the `HBaseClient` so that it can support multiple tables with the same connection would make sense here, instead of creating a separate `HBaseClient` for each Table.   This was originally [discussed here](https://github.com/apache/metron/pull/1458#discussion_r306954597).  There is a similar scenario in the `TaxiiHandler`.  

I'd like to do this work as a follow-on PR, rather than increasing the heft of this PR.